Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "rpg-system-2d"
+version = "0.1.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+bevy = { version = "0.5.0", features = ["dynamic"] }
```

Comment:
How about adding a build dependency (not so sure about the exact version, just added mine)?
```
[build-dependencies]
cmake = 3.21.2
```